scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
+// Meter
+// ==========================================================================
+meter {
+	-webkit-appearance: none;
+	width: 550px;
+	height: 25px;
+}
+
+.meter::-webkit-meter-bar, .multiColor::-webkit-meter-bar, .stripMeter::-webkit-meter-bar {
+	box-shadow: inset 0 0 7px -2px rgba(0,0,0,0.35);
+	background: #f1f1f1;
+}
+
+.meter::-webkit-meter-optimum-value {
+	background: #37bfc9;
+}
+
+
+.multiColor::-webkit-meter-optimum-value {
+	background-image: linear-gradient(
+		90deg, 
+		#3ee3ef 10%, 
```

Comment:
Since this seems to be based off of the Chassis 'blue' color, I suggest pulling in a variable with the Chassis Blue, and then lighten it with the Sass color functions to allow for easy customization.